deployment/terraform/resources/providers.tf

```diff
@@ -15,3 +15,25 @@ terraform {
 
 data "azurerm_client_config" "current" {
 }
+
+
+# Terraform stuff to include
+# 1. This provider
+# 2. Cosmos DB containers
+# 3. The AKS Node Pool
+# 4. The Kubernetes namespace, secrets
+
+# provider "kubernetes" {
```

Comment:
Still working through this. Just running into a bit of trouble with the various ways of doing `kubelogin` auth (azurecli vs service principal).

It looks like pctasks has its own way to get values out of terraform and into later deployment steps, that we could perhaps use here instead of introducing this new provider.